news.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./scripts.js">
```

Comment:
j/w

Reply:
j/w nie wiem co to znaczy owrapowac, owrapowac to ja moge cyke grubego w wrapa

Reply:
nie owrapowac, plik skryptow to nie plik styli, importujesz 2 razy ten sam, ten mozna wyrzucic